jobala/projectX
# Barcode scanner feature

#### What does this PR do?
This PR is an implementation of the Barcode scanner feature in the Mobile App
#### Description of Task to be completed?
`react-native-smart-barcode-scanner` library was to embedded in the app to give it this functionality.
#### How should this be manually tested?
- Clone the repo.
- cd into the project directory.
- checkout into the develop branch. 
- Install the dependencies using npm with `npm install`
- Ensure to have an emulator or phone connected, then build the application with `react-native run-android`
#### Any background context you want to provide?
The app didn't have a barcode scanner feature
